app/src/main/java/com/keylesspalace/tusky/components/compose/ComposeViewModel.kt

```diff
@@ -211,15 +213,8 @@ class ComposeViewModel @Inject constructor(
     }
 
     fun didChange(content: String?, contentWarning: String?): Boolean {
-
-        val textChanged = !(
-            content.isNullOrEmpty() ||
-                startingText?.startsWith(content.toString()) ?: false
-            )
-
-        val contentWarningChanged = showContentWarning.value &&
-            !contentWarning.isNullOrEmpty() &&
-            !startingContentWarning.startsWith(contentWarning.toString())
+        val textChanged = (content ?: "") != (startingText ?: "")
+        val contentWarningChanged = (contentWarning ?: "") != (startingContentWarning ?: "")
```

Comment:
There is an extension function `orEmpty()` which is nicer than `?: ""` imho

Reply:
Done.